lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -111,6 +111,11 @@ public function execute(InputInterface $input, OutputInterface $output)
         $path = $this->generateMigration($configuration, $input, $version, $up, $down);
 
         $output->writeln(sprintf('Generated new migration class to "<info>%s</info>" from schema differences.', $path));
+        $commandName = sprintf('%s:execute', $this->getApplication()->extractNamespace($this->getName()));
+        if (count($command = $this->getApplication()->find($commandName))) {
+          $output->writeln(sprintf('To run just this migration for testing purposes, you can use "<info>%s --up %s</info>"', $command->getName(), $version));
+        }
```

Comment:
> To execute just this migration to make sure it works, you can use...

Also, the `"` at the end probably aren't needed - the `<info>` should be enough to make it stand out.